.github/workflows/manual_release.yml

```diff
@@ -47,4 +47,4 @@ jobs:
           MOMENTO_DDB_LOCK_VERSION: ${{ steps.semrel.outputs.version }}
         uses: gradle/gradle-build-action@v2
         with:
-          arguments: publishToSonatype closeSonatypeStagingRepository
+          arguments: publishToSonatype closeAndReleaseStagingRepository
```

Comment:
What does this do?

Reply:
So `closeSonatypeStagingRepository` simply uploads to the repository and doesn't release. That's just for sanity check - and the screenshot that I shared on Slack. This wouldn't require us to manually release each type, and Nexus will automatically release it for us. This is how the Java SDK is also wired.

Reply:
Essentially the current one on `main` just uploads artifacts and requires a manual release, and the one in the PR is automated to do releases through CI action.